src/js/views/patients/worklist/bulk-edit/bulk-edit_views.js

```diff
@@ -270,7 +266,8 @@ const BulkEditActionsBodyView = View.extend({
     }
 
     const time = this.model.get('time');
-    const isDisabled = !this.model.get('date') || (this.model.get('dateMulti') && !time);
+    const isDisabled = (this.model.get('dateMulti') && !time) || !this.model.get('date');
```

Comment:
This was mostly for test coverage. But I may be able to rethink the check itself because it may be overkill.

If place after the or, `(this.model.get('dateMulti') && !time)` is never evaluated because `this.model.get('date')` is `null` when `dateMulti` is true. https://github.com/RoundingWell/care-ops-frontend/blob/develop/src/js/apps/patients/worklist/sidebar/bulk-edit-actions_app.js#L47

I _think_ this check could just be `const isDisabled = !time || !this.model.get('date');`

Reply:
it's like the first check needs to be `(!this.model.get('date') && !this.model.get('dateMulti'))

Reply:
This logic is correct for all of those scenarios. This switch just ensures that we hit the `(this.model.get('dateMulti') && !time)` check in the multiple dates scenarios. Otherwise that condition was never getting evaluated (though the result was still correct).